test/controllers/update/overseas.entity.presenter.controller.spec.ts

```diff
@@ -107,6 +108,19 @@ describe("OVERSEAS ENTITY PRESENTER controller", () => {
     });
   });
 
+  describe ("GET tests for remove journey", () => {
+
+    test(`renders the ${UPDATE_PRESENTER_PAGE} page for remove`, async () => {
+      mockGetApplicationData.mockReturnValueOnce({ [PresenterKey]: PRESENTER_OBJECT_MOCK, [EntityNumberKey]: "OE123457" });
+      const resp = await request(app).get(`${OVERSEAS_ENTITY_PRESENTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(REMOVE_OVERSEAS_ENTITY_PRESENTER_PAGE_TITLE);
+      expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+      expect(resp.text).toContain(saveAndContinueButtonText);
```

Comment:
we should also check (expect) that the correct back link is in the html in this GET test

Reply:
added test

Reply:
the back link should have the journey parameter on it so it should be 
${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL}${JOURNEY_REMOVE_QUERY_PARAM}